package.json

```diff
@@ -12,20 +12,23 @@
   "bugs": {
     "url": "https://github.com/csandman/chakra-react-select/issues"
   },
-  "type": "module",
   "exports": {
     ".": {
-      "import": "./dist/index.js",
-      "require": "./dist/index.cjs",
+      "module": "./dist/index.mjs",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js",
       "default": "./dist/index.js"
-    }
+    },
+    "./package.json": "./package.json"
   },
   "main": "dist/index.js",
+  "module": "dist/index.mjs",
   "files": [
     "dist"
   ],
   "scripts": {
     "build": "tsup",
+    "check-exports": "attw --pack .",
```

Comment:
I also added the package `@arethetypeswrong/cli` in order to more easily ensure that the exports are set up correctly after every change, as this is the real thing I've been trying to make sure of.